quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/update/QuarkusCli32to315UpdateIT.java

```diff
@@ -0,0 +1,66 @@
+package io.quarkus.ts.quarkus.cli.update;
+
+import static io.quarkus.test.util.QuarkusCLIUtils.getDependencies;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.apache.maven.model.Dependency;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.QuarkusCliRestService;
+import io.quarkus.test.util.DefaultQuarkusCLIAppManager;
+import io.quarkus.test.util.IQuarkusCLIAppManager;
+
+@Tag("quarkus-cli")
+public class QuarkusCli32to315UpdateIT extends AbstractQuarkusCliUpdateIT {
```

Comment:
I don't know why we should test the same thing fom 3.2 and 3.8. I would assume if it's work from 3.2 to 3.15 it will also work from 3.8 to 3.15.

So in my POV it's not needed have two same test class

Reply:
ok I agree with your assumption, I'll remove the 32 class